README.md

```diff
@@ -114,20 +120,23 @@ In general, every development board has the following basic features:
 
 > Now that we covered basics of development boards, let us talk about the changes made in the new design. 
 
-## Major Changes for 2022
-
-| Feature  |  SRA Board 2019  | SRA Board 2020 | SRA Board 2022 |
-|:----:|:-------:| :-----: | :-----: |
-|[12V to 5V](#7805-5v-linear-regulator-to-lm2596-buck-convertor)  | LM7805 Linear Regulator | LM2596 Buck Convertor | LM2576-S Buck Convertor |
-|[5V to 3.3V](#ld33-33v-to-ams1117)| LD33 | AMS1117-3.3 | AMS1117-3.3 |
-|[Reverse Voltage Protection](#reverse-voltage-protection-diodes-to-p-mosfet) | Diodes | P-MOSFET | Diodes |
-|[Motor Driver](#l298n-to-tb6612fng)| L298N| TB6612FNG | TB6612FNG |
-|[No. of Motor Channels](#motor-driver-modes)|2|4|4|
-|[No. of Switches](#moving-back-to-the-vintage-bar-graph-leds-and-more-switches)|2|4|4|
-|[No. of LEDs](#moving-back-to-the-vintage-bar-graph-leds-and-more-switches)|2|8|8|
-|[Over Current Protection](#protection-against-over-current)  | Bulky Glass Fuses | Bulky Glass Fuses | PTC Resettable Fuses|
-|[Sensor Port Connectors](#protection-against-over-current)  | FRC (Fused Ribbon Cable) Connectors | JST (Japan Solderless Terminal) Connectors | JST Connectors |
-|[Component Type and Board Size](#component-type-and-board-size)  | THT (Through Hole), Big | THT (Through Hole), Big | SMD(Surface Mount Device), Smaller|
+## Major Changes for 2023
+
+| Feature  |  SRA Board 2019  | SRA Board 2020 | SRA Board 2022 | SRA Board 2023 |
+|:----:|:-------:| :-----: | :-----: | :-----: |
+|[12V to 5V](#7805-5v-linear-regulator-to-lm2596-buck-convertor)  | LM7805 Linear Regulator | LM2596 Buck Converter | LM2576-S Buck Converter | LM2576-S Buck Converter |
+|[5V to 3.3V](#ld33-33v-to-ams1117)| LD33 | AMS1117-3.3 | AMS1117-3.3 | AMS1117-3.3 |
+|[Reverse Voltage Protection](#reverse-voltage-protection-diodes-to-p-mosfet) | Diodes | P-MOSFET | Diodes | Diodes |
+|[Line Sensing Arrays (LSA)](#lsa_2023) | Photodiodes| Photodiodes| Photodiodes| IR Sensors |
+|[Number of LSA Sensors](#num_of_sensors)| 4| 4| 4| 5 |
+|[Motor Driver](#l298n-to-tb6612fng)| L298N| TB6612FNG | TB6612FNG | TB6612FNG  |
+| [Stepper Motor Driver](#stepper-motor-driver-port-a4988) | - | - | - | A4988 |
+|[No. of Motor Channels](#motor-driver-modes)|2|4|4|4| 
```

Comment:
Add a row below mentioning `No. of Stepper Motor Channels` which is 1 for SRA Board 2023 and update the existing row as `No. of DC Motor Channels` which is 2 in SRA Board 2023